src/spells.cpp

```diff
@@ -637,6 +645,12 @@ bool Spell::playerSpellCheck(Player* player) const
 		return false;
 	}
 
+	if (player->getHealth() < static_cast<int32_t>(getHealthCost(player)) && !player->hasFlag(PlayerFlag_HasInfiniteMana)) {
```

Comment:
should be
```c++
if (player->getHealth() <= ...
```
because the player could die if the ammount is exactly the same as his current health is.

Reply:
How i didn't notice this? Thanks for finding it. Fixed.